packages/replay-next/pages/variables.css

```diff
@@ -49,7 +49,7 @@
   --primary-accent-hover: #0194ff;
 
   --secondary-accent: #d72451;
-  --secondary-accent-hover: #d72451;
+  --secondary-accent-hover: #c61351;
```

Comment:
This regressed a couple of months ago in #10049

Reply:
Thanks for this fix, I was just looking into it because of #10049